aries-site/src/pages/components/page.mdx

```diff
@@ -0,0 +1,170 @@
+
+<Example
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/page/SinglePageContent.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/FilterControls/FilterControls.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/FilterControls/FiltersContext.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/PageHeader/PageHeader.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/data/mockData/trainingDataSets.json",
+  ]}
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  width="100%" 
+  showResponsiveControls={['fullScreen']}
+  screenContainer={{ scale: 0.6 }}
+>
+  <SinglePageContent />
+</Example>
+
+### Full-width background color
+Use cases where an application needs a background to span the entire width of a page section 
+is achieved by setting `background={{ fill: "horizontal", ...additional properties }}` on 
+PageContent.
+
+This treatment is often seen on marketing sites where backgrounds help visually 
+block content sections. This might also be seen in a context such as an app-store catalog where 
+the hero section might have an image spanning.
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/page/PageContentExample.js"
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  width="100%" 
+  showResponsiveControls={['fullScreen']}
+  screenContainer={{ scale: 0.57 }}
+>
+  <PageBackGroundExample />
+</Example>
+
+## Variants 
+
+### Wide
+"Wide" is Page's default `kind`. The vast majority of pages in HPE applications will 
+likely use Page's "wide" variant. This page layout ensures consistent content alignment 
+and padding from section to section, page to page, and app to app.
+
+See [page layouts](/templates/page-layouts#page-kinds) for specifications 
+and use-case guidance.
+
+<Example 
+  showResponsiveControls={['fullScreen']}
+  docs="https://v2.grommet.io/page?theme=hpe"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js"
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  height="650px"
+  width="100%"
+  screenContainer={{ scale: 0.57 }}
+>
+  <PageWideExample  />
+</Example>
+
+
+### Narrow
+For use cases where the desire is to provide a user with singular, focussed content or objective, 
+such as editing field values of an object, or stepping through a [wizard-based task flow](/templates/wizard).
+A "narrow" page layout limits the page content's width and positions the content at the center 
```

Comment:
```suggestion
A "narrow" page layout limits the page content's width to `large` and positions the content at the center 
```

Reply:
Trying to differentiate from "Wide" since wide also limits the width of the page.